pipeline_steps/tempest.groovy

```diff
@@ -32,29 +32,27 @@ def tempest(){
   common.conditionalStage(
     stage_name: "Tempest Tests",
     stage: {
-      try{
-        def result = tempest_run()
-        def second_result = ""
-        if(result.contains("Race in testr accounting.")){
-          second_result = tempest_run()
-        }
-        if(second_result.contains("Race in testr accounting.")) {
-          currentBuild.result = 'FAILURE'
-        }
+      retry(2){
+        try{
+          def result = tempest_run()
+          if(result.contains("Race in testr accounting.")) {
+            currentBuild.result = 'FAILURE'
```

Comment:
Let's keep the testr check in place then @mkam.  So if we throw an error in this if block will that be sufficient?